bench-vortex/src/bin/clickbench.rs

```diff
@@ -208,7 +208,14 @@ fn main() -> anyhow::Result<()> {
         .targets
         .iter()
         .any(|t| t.engine() == Engine::DuckDB)
-        .then(|| ddb::build_and_get_executable_path(&args.duckdb_path, false));
+        .then(|| {
+            let path = ddb::get_executable_path(&args.duckdb_path);
+            // If the path is to vortex duckdb, try to rebuild
+            if args.duckdb_path.is_none() {
+                ddb::build_executable_path();
```

Comment:
We just build DuckDB here, right? Probably `build_executable_path` should be renamed to `build_duckdb` or so.